numpy/random/src/xoshiro256/xoshiro256.c

```diff
@@ -29,7 +29,7 @@ extern NPY_INLINE uint32_t xoshiro256_next32(xoshiro256_state *state);
 
 void xoshiro256_jump(xoshiro256_state *state)
 {
-	int i, b;
+	unsigned i, b;
```

Comment:
Given the comparison against sizeof below, size_t would be most appropriate (stddef.h)

Reply:
We only need them to be unsigned to avoid the warning, but sure, might as well make them consistent with `sizeof`.

Reply:
Done.